packages/react-i18n-universal-provider/README.md

```diff
@@ -66,7 +71,7 @@ npx yarn-deduplicate --packages @shopify/react-effect yarn.lock
 ```
 
 ```bash
-$ yarn why @shopify/react-i18n # ensure no duplicate / unmet dependencies 
-yarn list  # ensure no duplicate / unmet dependencies 
+$ yarn why @shopify/react-i18n # ensure no duplicate / unmet dependencies
+yarn list  # ensure no duplicate / unmet dependencies
```

Comment:
just to double check, is this file all stylistic update?

Reply:
Yeah, I didn't actually change anything myself; my editor just auto-linted the code on save.